demos/more_examples/simple/table_to_graph_three_ways.ipynb

```diff
@@ -0,0 +1,902 @@
+      ],
+      "source": [
+        "g2 = graphistry.hypergraph(df, ['attackerIP', 'victimIP', 'vulnName'], direct=True)['graph']\n",
+        "g2.plot()"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "sXrMttyum_qK"
+      },
+      "source": [
+        "We can control many aspects. In this case:\n",
+        "\n",
+        "* Causally directed edges: attackerIP->victimIP, attackerIP->vulnName, vulnName->attackerIP\n",
+        "\n",
+        "* Combine name spaces: When an IP appears both as a victimIP and attackerIP, collapse into one node, vs treating those columns as distinct node ID namespaces"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 5,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 595
+        },
+        "id": "9Xy1QhXjfU0x",
+        "outputId": "49b25a5a-dfb8-422b-e1ef-70b2a46ac9b0"
+      },
+      "outputs": [
+        {
+          "name": "stdout",
+          "output_type": "stream",
+          "text": [
+            "# links 660\n",
+            "# events 220\n",
+            "# attrib entities 212\n"
+          ]
+        },
+        {
+          "data": {
+            "text/html": [
+              "\n",
+              "            <iframe id=\"622f9db3-2857-4a80-b790-be882ddaf601\" src=\"https://hub.graphistry.com/graph/graph.html?dataset=c1bccf8762ea4ced902624dc2c26612b&type=arrow&viztoken=d70114f6-6fe4-4ffe-80f5-d45790f8a6a3&usertag=fe5494cc-pygraphistry-0.35.4+66.g9a3a886&splashAfter=1736535062&info=true\"\n",
+              "                    allowfullscreen=\"true\" webkitallowfullscreen=\"true\" mozallowfullscreen=\"true\"\n",
+              "                    oallowfullscreen=\"true\" msallowfullscreen=\"true\"\n",
+              "                    style=\"width:100%; height:500px; border: 1px solid #DDD; overflow: hidden\"\n",
+              "                    \n",
+              "            >\n",
+              "            </iframe>\n",
+              "        \n",
+              "            <script>\n",
+              "                try {\n",
+              "                  $(\"#622f9db3-2857-4a80-b790-be882ddaf601\").bind('mousewheel', function(e) { e.preventDefault(); });\n",
+              "                } catch (e) { console.error('exn catching scroll', e); }\n",
+              "            </script>\n",
+              "        "
+            ],
+            "text/plain": [
+              "<IPython.core.display.HTML object>"
+            ]
+          },
+          "execution_count": 5,
+          "metadata": {},
+          "output_type": "execute_result"
+        }
+      ],
+      "source": [
+        "g2b = graphistry.hypergraph(df, ['attackerIP', 'victimIP', 'vulnName'], direct=True, opts={\n",
+        "    'EDGES': {\n",
+        "        'attackerIP': ['victimIP', 'vulnName'],\n",
+        "        'vulnName': ['victimIP']\n",
+        "    }\n",
```

Comment:
needs comma here. python complains about invalid object

Reply:
it runs in ipynb so think so, more of an issue for json parsers

Reply:
done in another pr, tx for catch!